src/Features/CSharp/Portable/Completion/KeywordRecommenders/GlobalKeywordRecommender.cs

```diff
@@ -33,11 +33,8 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
 
             return
                 context.IsTypeContext ||
-                UsingKeywordRecommender.IsUsingDirectiveContext(context, forGlobalKeyword: true, cancellationToken) ||
                 context.IsAnyExpressionContext ||
-                syntaxTree.IsAfterKeyword(position, SyntaxKind.ConstKeyword, cancellationToken) ||
-                syntaxTree.IsAfterKeyword(position, SyntaxKind.RefKeyword, cancellationToken) ||
-                syntaxTree.IsAfterKeyword(position, SyntaxKind.ReadOnlyKeyword, cancellationToken) ||
```

Comment:
These are already part of `IsTypeContext`:

https://github.com/dotnet/roslyn/blob/2bc0eb37e79e7379671be5de2a2eec61cb729283/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs#L724-L726